suite-native/coin-enabling/src/hooks/useCoinEnabling.tsx

```diff
@@ -0,0 +1,39 @@
+import { useDispatch, useSelector } from 'react-redux';
+
+import { NetworkSymbol } from '@suite-common/wallet-config';
+import { FeatureFlag, useFeatureFlag } from '@suite-native/feature-flags';
+import {
+    selectAreTestnetsEnabled,
+    selectEnabledDiscoveryNetworkSymbols,
+    selectDiscoverySupportedNetworks,
+    toggleEnabledDiscoveryNetworkSymbols,
+} from '@suite-native/discovery';
+import { DeviceRootState } from '@suite-common/wallet-core';
+
+export const useCoinEnabling = () => {
+    const dispatch = useDispatch();
+    const [isCoinEnablingActive] = useFeatureFlag(FeatureFlag.IsCoinEnablingActive);
+    const areTestnetsEnabled = useSelector(selectAreTestnetsEnabled);
+    const availableNetworks = useSelector((state: DeviceRootState) =>
+        selectDiscoverySupportedNetworks(state, areTestnetsEnabled),
+    );
+    const enabledNetworkSymbols = useSelector(selectEnabledDiscoveryNetworkSymbols);
+
+    const applyDiscoveryChanges = () => {
+        // TODO: remove disabled network accounts and run discovery check
+    };
+
+    const toggleEnabledDiscoveryNetworkSymbol = ({
+        networkSymbol,
+    }: {
+        networkSymbol: NetworkSymbol;
+    }) => dispatch(toggleEnabledDiscoveryNetworkSymbols(networkSymbol));
```

Comment:
Is this just renaming? Or the point here is to save importing dispatch? 

Considering the almost identical names (where `toggleEnabledDiscoveryNetworkSymbols` is confusing because it has only one symbol as a parameter), it seems to me that it makes the situation rather confusing. How about just renaming the action?

Reply:
roger 🫡  :)

8fe7abad8d